src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java

```diff
@@ -279,8 +278,26 @@ public static String getGeometryProperty(final Geometry geometry, final String k
         }
         else
         {
-            return propertyMap.get(key);
+            result.putAll(propertyMap);
```

Comment:
If propertyMap is null, and we return null, do we have to worry about NullPointerExceptions in the use cases below?  Would it be better to return the empty Map?